packages/ui/src/app/pages/Forum/components/ForumTabs.tsx

```diff
@@ -23,7 +23,7 @@ export const ForumForumTabs = ({ categoryCount }: ForumForumTabsProps) => {
   const tabs = usePageTabs([
     ['Categories', ForumRoutes.forum, { count: categoryCount }],
     ['Latest threads ', ForumRoutes.latestThreads],
-    ['Top threads', ForumRoutes.topThreads],
+    // ['Top threads', ForumRoutes.topThreads],
```

Comment:
Great PR thank you :pray: 

Please just uncomment this, because thanks to #3173 we can keep this tab for now.

```suggestion
    ['Top threads', ForumRoutes.topThreads],
```